src/Components/Admin/admin.js

```diff
@@ -111,6 +112,17 @@ export default function Admin() {
     const [value, setValue] = React.useState("rooms");
     const [anchorEl, setAnchorEl] = React.useState(null);
     const [createModalOpen, setCreateModalOpen] = React.useState(false);
+    const [environments, setEnvironments] = React.useState([]);
+
+    const getAllEnvironments = async () => {
+        const data = await getAllScenes();
+        setEnvironments(data);
+    };
+
+    useEffect(() => {
+        getAllEnvironments();
+    }, []);
```

Comment:
Correct me if I'm wrong, but this will get called all the time right? Should we only call it when that tab of the page is opened?

Reply:
Good point. Fixed.